python/nav/web/seeddb/forms/__init__.py

```diff
@@ -372,6 +372,8 @@ class DeviceGroupForm(forms.ModelForm):
     netboxes = forms.ModelMultipleChoiceField(
         queryset=Netbox.objects.all(), required=False
     )
+    netboxes.widget.attrs.update({"class": "select2"})
+    no_crispy = True
```

Comment:
When crispy is gone we will no longer need this for choosing which form to show. It is still useful to hold extra form attributes though, like with the filter forms.